MushroomObserver/mushroom-observer
# SpeciesList year

Prevent users from creating a SpeciesList with a nil `when`.

- Uses Observation validation logic to validate `SpeciesList.when`
- Extract that logic to a Concern, using the concern in both Observation and SpeciesList
- Delivers https://www.pivotaltracker.com/story/show/177682975

### Manual Test 
- Create a SpeciesList, blanking out the Year in When.
Expected Result: It should create and display the list, with When set to the date the list was created
